clients/da-vinci-client/src/main/java/com/linkedin/davinci/helix/HelixParticipationService.java

```diff
@@ -339,13 +338,10 @@ private void asyncStart() {
 
     VeniceServerConfig veniceServerConfig = veniceConfigLoader.getVeniceServerConfig();
     VeniceProperties veniceProperties = veniceServerConfig.getClusterProperties();
-    PubSubProducerAdapterFactory pubSubProducerAdapterFactory =
```

Comment:
Same: This change creates a new PubSubClientsFactory object which is not desirable IMHO. Besides, in https://github.com/linkedin/venice/pull/1705 we're removing VW instantiation here altogether.